### ISSUE TITLE
Kibana status is Yellow - "plugin:opendistro_security@6.5.4 Tenant indices migration failed"

### ISSUE THREAD

**Issue by aetter** [opendistro-for-elasticsearch/security-kibana-plugin#1]
Originally reported in https://github.com/opendistro/for-elasticsearch-docs/issues/6:

```
Using the sample docker-compose.yml (https://opendistro.github.io/for-elasticsearch-docs/docs/install/docker/).

then run:
docker-compose up
Able to login to Kibana and everything works fine.

then:
docker-compose stop
docker-compose start

Able to login to Kibana but see error:
Kibana status is Yellow
plugin:opendistro_security@6.5.4 Tenant indices migration failed

Unable to do anything in Kibana. Any help would be appreciated.
```

**Comment by gabrielcalderon** [comment]
It seems like the fix for this issue was already merged. When can we expect a new release of the Opendistro Kibana ?

**Comment by andrx** [comment]
if you still want to check it without this plugin add this to `/etc/kibana/kibana.yml`

	opendistro_security.multitenancy.enabled: false

**Comment by gabrielcalderon** [comment]
@andrx Thanks, I will give that a try.

**Comment by Lock128** [comment]
When will this new version 0.7.0.1 be built into a new docker container?

**Comment by gaurav6041** [comment]
Yes i also need it. Can you guys update when will this be built into a new docker

**Comment by hardik-k-shah** [comment]
New docker image and RPM (0.7.1) was built and released to address this issue. 
More details: https://opendistro.github.io/for-elasticsearch-docs/#version-history

**Comment by MysterionRise** [comment]
Using docker image of 0.9.0 version, still face the same error - any ideas why? Workaround with disabling tenancy worked - but that's kinda dirty hack.

**Comment by devansvd** [comment]
I can confirm the same, 0.9.0 version still face the same error. Is this resolved ?

`{"type":"error","@timestamp":"2019-06-01T06:28:21Z","tags":["error","migration"],"pid":1,"level":"error","error":{"message":"Authorization Exception: Authorization Exception","name":"Error","stack":"Authorization Exception :: {\"path\":\"/_opendistro/_security/tenantinfo\",\"query\":{},\"statusCode\":403,\"response\":\"\"}\n    at respond (/usr/share/kibana/node_modules/elasticsearch/src/lib/transport.js:308:15)\n    at checkRespForFailure (/usr/share/kibana/node_modules/elasticsearch/src/lib/transport.js:267:7)\n    at HttpConnector.<anonymous> (/usr/share/kibana/node_modules/elasticsearch/src/lib/connectors/http.js:166:7)\n    at IncomingMessage.wrapper (/usr/share/kibana/node_modules/elasticsearch/node_modules/lodash/lodash.js:4935:19)\n    at IncomingMessage.emit (events.js:194:15)\n    at endReadableNT (_stream_readable.js:1103:12)\n    at process._tickCallback (internal/process/next_tick.js:63:19)"},"message":"Authorization Exception: Authorization Exception"}
{"type":"log","@timestamp":"2019-06-01T06:28:21Z","tags":["status","plugin:opendistro_security@6.7.1","info"],"pid":1,"state":"yellow","message":"Status changed from yellow to yellow - Tenant indices migration failed","prevState":"yellow","prevMsg":"Setting up index template."}
`

**Comment by devansvd** [comment]
> I can confirm the same, 0.9.0 version still face the same error. Is this resolved ?
> 
> `{"type":"error","@timestamp":"2019-06-01T06:28:21Z","tags":["error","migration"],"pid":1,"level":"error","error":{"message":"Authorization Exception: Authorization Exception","name":"Error","stack":"Authorization Exception :: {\"path\":\"/_opendistro/_security/tenantinfo\",\"query\":{},\"statusCode\":403,\"response\":\"\"}\n at respond (/usr/share/kibana/node_modules/elasticsearch/src/lib/transport.js:308:15)\n at checkRespForFailure (/usr/share/kibana/node_modules/elasticsearch/src/lib/transport.js:267:7)\n at HttpConnector.<anonymous> (/usr/share/kibana/node_modules/elasticsearch/src/lib/connectors/http.js:166:7)\n at IncomingMessage.wrapper (/usr/share/kibana/node_modules/elasticsearch/node_modules/lodash/lodash.js:4935:19)\n at IncomingMessage.emit (events.js:194:15)\n at endReadableNT (_stream_readable.js:1103:12)\n at process._tickCallback (internal/process/next_tick.js:63:19)"},"message":"Authorization Exception: Authorization Exception"} {"type":"log","@timestamp":"2019-06-01T06:28:21Z","tags":["status","plugin:opendistro_security@6.7.1","info"],"pid":1,"state":"yellow","message":"Status changed from yellow to yellow - Tenant indices migration failed","prevState":"yellow","prevMsg":"Setting up index template."} `

The above issue resolved for me by using the `kibanaserver` in config.yml and kibana.yml `elasticsearch.username: kibanaserver`. While running kibana docker don't pass elastic username it overrides the kibana.yml username settings.

**Comment by MysterionRise** [comment]
@devansvd could you come up with proposed PR to have this as a default settings?

**Comment by devansvd** [comment]
@MysterionRise don't start odfe-kibana docker container with any elastic username. It overrides the elastic username specified in kibana.yml. I can confirm the issue is resolved for me in docker 0.9.0 version

**Comment by sadafnoor-augmedix** [comment]
plugin:opendistro_security@7.0.1 | Tenant indices migration failed
